miniapps/shifted/integ_algoim.hpp

```diff
@@ -20,6 +20,8 @@
 namespace mfem
 {
 
+
+/*
```

Comment:
Delete instead of commenting out.

Reply:
I will delete the whole file - it is not need anymore.

Reply:
fixed